src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public ColorsForBall getRandomColor() {
+        int colorForBallRandom = new Random().nextInt(ColorsForBall.values().length);
```

Comment:
We can add field with type `Random` to our class, so we won't need to create an instance every time when method invoked